substrate/client/rpc/src/testing.rs

```diff
@@ -20,11 +20,50 @@
 
 use std::{future::Future, sync::Arc};
 
-use sp_core::testing::TaskExecutor;
+/// A task executor that can be used for running RPC tests.
+///
+/// Warning: the tokio runtime must be initialized before calling this.
+#[derive(Clone)]
+pub struct TokioTestExecutor(tokio::runtime::Handle);
```

Comment:
nit: I do remember running into something similar in another PR and needing a custom executor, but can't quite remember. Was `sp_core::TaskExecutor` replaced by tokio to play nice with channels?

Reply:
It's not needed anymore but before we use `tokio::mpsc::send_timeout` which only works on the tokio runtime.

I can revert it but I was too lazy to do it.

Reply:
That should be fine, no need to revert it imo. Thanks for the answer!